packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -15,21 +40,46 @@ export const AdvancedSettings = ({
 }: {
   destinationAddress?: string
   onChange: (value?: string) => void
-  error: AdvancedSettingsErrors | null
+  error: DestinationAddressErrors | null
 }) => {
   const {
-    app: { selectedToken }
+    app: { selectedToken, isDepositMode }
   } = useAppState()
+  const { l1, l2 } = useNetworksAndSigners()
   const { isEOA = false, isSmartContractWallet = false } = useAccountType()
 
   const [collapsed, setCollapsed] = useState(true)
+  const [warning, setWarning] = useState<string | null>(null)
 
   useEffect(
     // Show on page load if SC wallet since destination address mandatory
     () => setCollapsed(!isSmartContractWallet),
     [isSmartContractWallet]
   )
 
+  useEffect(() => {
```

Comment:
As per previous discussions, we want to show a warning for EOA if a contract address is specified in the destination address input.

Reply:
are we not detecting that it's a known token address & showing the user a more specific error message?

Reply:
https://github.com/OffchainLabs/arbitrum-token-bridge/issues/922

Will be part of this